rust/tokens_transfer/README.md

```diff
@@ -0,0 +1,71 @@
+# Tokens Transfer
+
+Tokens Transfer is a canister that can transfer token from its account to other accounts.
+It is an example of a canister that uses the Ledger canister.
+
+
+## Interface
+
+2. `transfer`: takes in input the amount of tokens to transfer, the account (and optionally the subaccount) to which to transfer the tokens and returns either success or an error in case e.g. the tokens transfer canister doesn't have enough tokens to do the transfer. In case of success, a unique identifier of the transaction is returned. This identifier will be stored in the memo of the transaction in the Ledger.
+
+
+## Initialization
+
+The canister expects three arguments:
+1. `ledger_canister_id`: the canister id of the ledger canister
+2. `subaccount`: the optional subaccount of the canister account from which tokens will be withdrawn
+3. `transaction_fee`: a constant representing the transaction fee of the ledger
+
+
+## Test Locally
+
+1. start the local replicate with `dfx start`
+2. deploy the Ledger canister locally. Add some tokens to your account (`dfx identity get-principal`) in the initialization parameters of the Ledger canister.
+```bash
+# MINTING_ACCOUNT_ID_HEX and ACCOUNT_ID_HEX are the hex representation
+# of the minting principal and your principal respectively
+read -r -d '' ARGS <<EOM
+(record {
+     minting_account="${MINTING_PRINCIPAL_HEX}";
+     initial_values=vec { record { "${YOUR_PRINCIPAL_HEX}"; record { e8s=10_000_000_000 } }; };
```

Comment:
Are you sure those are _principals_ and  not _accounts ids_?

Reply:
Changed to account ids.